src/main/java/io/asyncer/r2dbc/mysql/client/ReactorNettyClient.java

```diff
@@ -287,9 +287,6 @@ private void handleClose() {
     }
 
     @SuppressWarnings("unchecked")
```

Comment:
I don't think we need this line any longer

Reply:
fixed